### ISSUE TITLE
These formats give me 'invalid time'

### ISSUE THREAD

**Issue by ryanwilliamquinn** [zackdever/time#9]
4:59:30 pm
09:42:43am
0:04:01

Looks like it doesn't support seconds?

**Comment by zackdever** [comment]
I don't think it does, but should be easy enough to add. I haven't touched this in a couple years, but I should dust it off and look at the open PR and other issues. I'll take a look this evening.

**Comment by ryanwilliamquinn** [comment]
Thanks, much appreciated.
